packages/contracts/contracts/core/permission/PermissionManager.sol

```diff
@@ -344,7 +344,10 @@ contract PermissionManager is Initializable {
     /// @notice A modifier to be used to check permissions on a target contract.
     /// @param _where The address of the target contract for which the permission is required.
     /// @param _permissionId The permission identifier required to call the method this modifier is applied to.
-    function _auth(address _where, bytes32 _permissionId) private {
+    function _auth(address _where, bytes32 _permissionId) private view {
+        // TODO: will we need to emit `who` as well ? below `who: msg.sender` 
+        // is not actul who that we grant the permission in the end, it's the sender
+        // that initiations permission.
```

Comment:
```suggestion
```
Why isn't it the sender that asks for authorization?
`msg.sender` is the address asking the `PermissionManager` to do something for him/her.